semgrep/semgrep/target_manager.py

```diff
@@ -208,20 +208,6 @@ def yield_verbose_lines(self) -> Iterator[Tuple[Literal[0, 1, 2], str]]:
         else:
             yield 2, "<none>"
 
-        yield 1, "Skipped by --include patterns:"
-        if self.cli_includes:
-            for path in self.cli_includes:
-                yield 2, with_color(Colors.cyan, str(path))
-        else:
-            yield 2, "<none>"
-
-        yield 1, "Skipped by --exclude patterns:"
-        if self.cli_excludes:
-            for path in self.cli_excludes:
-                yield 2, with_color(Colors.cyan, str(path))
-        else:
-            yield 2, "<none>"
-
```

Comment:
I think you removed the wrong lines here. This looks like logs about the global CLI flags `--include` and `--exclude`. The stuff we wanted to get rid of is the per-rule patterns which are printed for each rule below under `        for rule_id in self.rule_ids_with_skipped_paths:`.